sabnzbd/interface.py

```diff
@@ -282,6 +282,11 @@ def index(self, **kwargs):
             cfg.warned_old_queue.set(True)
             config.save_config()
             return panic_old_queue()
+        
+        if not cfg.notified_new_skin() and cfg.web_dir() != 'Glitter':
+            logging.warning(T('Try our new skin Glitter! Fresh new design that is optimized for desktop and mobile devices. Go to Config -> General to change your skin.'))
+            cfg.notified_new_skin.set(True)
+            config.save_config()
```

Comment:
Move "cfg.notified_new_skin.set(True)" below the "if" statement so that it is always executed.
Otherwise you'll also get the message when going from Glitter to Plush.